### DISCUSSION TITLE
How to Pay for ChistaDATA Cloud?

### DISCUSSION THREAD

**Discussion by emoido** [ChistaDATA/Cloud#4] [❓FAQ] [answered]
How to Pay for ChistaDATA Cloud?

**Comment by TheraniAA** [comment, marked as answer]
Sign up for an account on the ChistaDATA website and select **Setting -> Billing** from the left-hand menu and select Upgrade Plan to choose the service plan that suits your needs. After choosing your desired plan, you will be redirected to a payment processing platform known as Stripe.
